server/plugin.go

```diff
@@ -72,6 +82,30 @@ func (p *Plugin) OnActivate() error {
 		return fmt.Errorf("could not open target tsdb: %w", err)
 	}
 
+	scrapeInterval := *p.configuration.ScrapeIntervalSeconds
+
+	// TODO(isacikgoz): Use multiple targets for HA env
+	ls := labels.FromMap(map[string]string{
+		model.AddressLabel:        "localhost:8067",
+		model.ScrapeIntervalLabel: fmt.Sprintf("%ds", scrapeInterval),
+		model.ScrapeTimeoutLabel:  fmt.Sprintf("%ds", *p.configuration.ScrapeTimeoutSeconds),
+	})
+	lb := labels.NewBuilder(ls)
+
+	lset, origLabels, err := scrape.PopulateLabels(lb, &config.ScrapeConfig{
+		JobName: "prometheus",
+	}, true)
+	if err != nil {
+		return err
+	}
+
+	target := scrape.NewTarget(lset, origLabels, url.Values{})
+
+	// Mutator is being used to apply labels to the metric samples
+	sampleMutator = func(l labels.Labels) labels.Labels {
```

Comment:
I see this is not being used anywhere. Were you planning to add code later?

Reply:
Yes, it will be used later on, this was included in this PR for demonstration of the labels being provided.